packages/genesys-spark/scripts/build-token-files.mjs

```diff
@@ -0,0 +1,64 @@
+#! /usr/bin/env node
+import { globSync } from 'glob';
+import fs from 'fs';
+import path from 'path';
+import { dirname } from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+const FOLDER = process.env.OUTDIR || 'dist';
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+const projectPath = path.join(__dirname, '..');
+
+const cssFileFolder = path.join(projectPath, `${FOLDER}/css/flare`);
+const cssFilesGlob = path.join(cssFileFolder, '*.css');
+
+const cssFiles = globSync(cssFilesGlob);
+
+function writeToDistTokensFolder(filename, fileContent) {
+  const folder = path.join(projectPath, `${FOLDER}/beta/tokens/flare`);
```

Comment:
These will only be in a beta folder until I get confirmation from a team using them that they are ok. I would say a week or two at most.